EIPS/eip-4844.md

```diff
@@ -45,17 +45,18 @@ Compared to full data sharding, this EIP has a reduced cap on the number of thes
 | `BLOB_COMMITMENT_VERSION_KZG` | `Bytes1(0x01)` |
 | `POINT_EVALUATION_PRECOMPILE_ADDRESS` | `Bytes20(0x14)` |
 | `POINT_EVALUATION_PRECOMPILE_GAS` | `50000` |
-| `MAX_BLOBS_PER_BLOCK` | `16` |
-| `TARGET_BLOBS_PER_BLOCK` | `8` |
-| `MAX_BLOBS_PER_TX` | `2` |
-| `GASPRICE_UPDATE_FRACTION_PER_BLOB` | `64` |
+| `MAX_DATA_GAS_PER_BLOCK` | `2**21` |
```

Comment:
@SamWilsn  and @lightclient  can we quickly allowlist the `https://github.com/ethereum/consensus-specs`which I think was a consensus?

Reply:
@Pandapip1 I see you approved the PR but the merge is failing because of the links to the CL spec. Should we force-merge this?

Reply:
Yes

Reply:
Do you have this power :-D ?